packages/appstore/src/modules/onboarding/onboarding.scss

```diff
@@ -2,7 +2,7 @@
     display: flex;
     flex-direction: column;
     gap: 3rem;
-    background-color: var(--general-hover);
+    background-color: var(--general-section-1);
```

Comment:
@arshad-rao-deriv This change make the background darker for both light and dark theme, but in the description it's just mentioned change it for dark theme.
If there is no color with the appropriate set of colors, you need to define one new variable in the theme.scss file.

Reply:
@mahdiyeh-deriv The variable --general-section-1 for light theme is color-grey-2: #f2f3f4 and for dark theme it's color-black-3: #151717 and these are the hex values defined for dark and light theme in figma designs.